src/_utils.ts

```diff
@@ -85,10 +112,13 @@ export function setupOutgoing(outgoing, options, req, forward?) {
   const target = options[forward || "target"];
   const targetPath =
     target && options.prependPath !== false
-      ? target.pathname || target.path || ""
+      ? (target as URL).pathname ||
+        // NOTE: http-proxy receives url.Url instead of URL
+        (target as Partial<url.Url>).path ||
```

Comment:
Since httpxy receives `URL` instead of the legacy `Url`, this line can be removed.
```suggestion
```